src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/UserAuthDetails.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+public class UserAuthDetails {
```

Comment:
not keen to move this to the idam client?

Reply:
another PR ... ;)